code/controllers/subsystem/dbcore.dm

```diff
@@ -49,6 +49,8 @@ SUBSYSTEM_DEF(dbcore)
 	var/db_daemon_started = FALSE
 
 /datum/controller/subsystem/dbcore/Initialize()
+	Connect()
```

Comment:
:thonk:

why? Pretty sure clients getting created cause connects before this can ever happen

Reply:
I moved the init of dbcore up so that it also shuts down last.

This is just in case no clients connect, so that the schema error msgs still appear.